adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -56,43 +53,45 @@ - (void)renderBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
     NSError *error = GADMAdapterPangleErrorWithCodeAndDescription(
         GADPangleErrorInvalidServerParameters,
         [NSString stringWithFormat:@"%@ cannot be nil.", GADMAdapterPanglePlacementID]);
-    _loadCompletionHandler(nil, error);
+    self.loadCompletionHandler(nil, error);
     return;
   }
 
   NSError *error = nil;
   PAGBannerAdSize bannerSize = [self bannerSizeFormGADAdSize:adConfiguration.adSize error:&error];
   if (error) {
-    _loadCompletionHandler(nil, error);
+    self.loadCompletionHandler(nil, error);
     return;
   }
 
   PAGBannerRequest *request = [PAGBannerRequest requestWithBannerSize:bannerSize];
   request.adString = adConfiguration.bidResponse;
-
-  GADPangleRTBBannerRenderer *__weak weakSelf = self;
+  __weak typeof(self) weakSelf = self;
   [PAGBannerAd loadAdWithSlotID:placementId
                         request:request
-              completionHandler:^(PAGBannerAd *_Nullable bannerAd, NSError *_Nullable loadError) {
-                GADPangleRTBBannerRenderer *strongSelf = weakSelf;
-
-                if (loadError) {
-                  strongSelf->_loadCompletionHandler(nil, loadError);
-                  return;
-                }
-
-                CGRect frame = bannerAd.bannerView.frame;
-                frame.size = bannerSize.size;
-                bannerAd.bannerView.frame = frame;
-                bannerAd.rootViewController = adConfiguration.topViewController;
-
-                strongSelf->_bannerAd = bannerAd;
-                strongSelf->_bannerAd.delegate = strongSelf;
-
-                if (strongSelf->_loadCompletionHandler) {
-                  strongSelf->_delegate = strongSelf->_loadCompletionHandler(strongSelf, nil);
-                }
-              }];
+              completionHandler:^(PAGBannerAd * _Nullable bannerAd,
+                                  NSError * _Nullable loadError) {
+    __strong typeof(weakSelf) strongSelf = weakSelf;
```

Comment:
Can you update this to `GADPangleRTBBannerRenderer *strongSelf = weakSelf;`? The same applies to the other ad formats.